cmd/distributed/roles/roles.go

```diff
@@ -76,8 +77,11 @@ func (mr *MilvusRoles) Run(localMsg bool) {
 
 	if mr.EnableMaster {
 		var ms *components.MasterService
+		var wg sync.WaitGroup
 
+		wg.Add(1)
 		go func() {
+			defer wg.Done()
```

Comment:
call wg.Done()  after ms has been assigned, not `defer wg.Done()`, this go routine might not return

Reply:
ok